draft-ietf-rtcweb-jsep.xml

```diff
@@ -273,12 +273,11 @@ ipr="trust200902">
         what the local side is willing to receive, which, when
         intersected with the set of codecs the remote side supports,
         specifies what the remote side should send. However, not all
-        parameters follow this rule; for example, the fingerprints
-        <xref target="RFC8122"></xref> sent to a
-        remote party are calculated based on the local certificate(s)
-        offered; the remote party MUST either accept these parameters
-        or reject them altogether, with no option to choose different
-        values.</t>
+        parameters follow this rule, and the remote side MUST either
```

Comment:
This sentence still feels confusing to me, i.e., it's not clear what 'those' should refer to here.

Suggest rewriting as 
> ... not all parameters follow this rule; some parameters are declarative, and the remote side must either accept them or reject them altogether.